comment/views.py

```
@@ -48,126 +48,120 @@ def get(self, request):
+                ### 🔻 이 부분 수정 🔻 ###
+        if not request.user.is_authenticated:
+            return Response(
+                {"detail": "please signin"}, status=status.HTTP_401_UNAUTHORIZED
+            )
+        author = request.user
+        ### 🔺 이 부분 수정 🔺 ###
+        content = request.data.get("content")
+
         try:
             comment = Comment.objects.get(id=comment_id)
         except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)